contents/blog/dogfooding.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2021-08-27
+title: The importance of dogfooding - Why product managers should use their product as much as their users
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../images/blog/dogfooding.png
+featuredImageType: full
+---
+
+Product testing is essential, but your testing won’t always pick up the small sources of friction which build up over time and frustrate your users. This is where ‘dogfooding’ comes in - regularly using your product to get the full picture and build empathy with your users.
+
+## How empathy improves prioritization
+It’s logical to deprioritize a “small UX bug” when you have big stratic new features to implement - but the only way to ensure that “small UX bug” doesn’t lead to high churn for your product is to put yourself in your users’ shoes. Doing this over time leads to better feature prioritization and more effective product decisions.
```

Comment:
Typo: stratic

Reply:
Saw that - wasn't sure if @marcushyett-ph meant any of [these definitions](https://www.definitions.net/definition/stratic).